### ISSUE TITLE
recaptcha-plugin HCaptcha support

### ISSUE THREAD

**Issue by hgenchev99** [berstend/puppeteer-extra#177]
There is a new captcha provider https://www.hcaptcha.com/ and some sites are moving to them. It will be nice to be able to parse them as well.

**Comment by vallovic** [comment]
Considering this, having hCaptcha support would be a great thing

https://blog.cloudflare.com/moving-from-recaptcha-to-hcaptcha/

**Comment by brunogaspar** [comment]
Pull requests to add this are very welcomed :)

**Comment by phillipalexander** [comment]
Would love to see this functionality added.

**Comment by berstend** [comment]
@phillipalexander hCaptcha support has landed (#383) 😄 

I opted to re-use the existing APIs so just upgrade to `puppeteer-extra-plugin-recaptcha@3.3.0` and `page.solveRecaptchas()` will take care of those:

```
reCAPTCHA v2, reCAPTCHA v3, invisible reCAPTCHA, hCaptcha, invisible hCaptcha
```

Note: Playwright support (#303) is right around the corner and will support solving captchas in Firefox & Webkit as well. :-)

**Comment by seuaCoder** [comment]
> @phillipalexander hCaptcha support has landed (#383) 😄
> 
> I opted to re-use the existing APIs so just upgrade to `puppeteer-extra-plugin-recaptcha@3.3.0` and `page.solveRecaptchas()` will take care of those:
> 
> ```
> reCAPTCHA v2, reCAPTCHA v3, invisible reCAPTCHA, hCaptcha, invisible hCaptcha
> ```
> 
> Note: Playwright support (#303) is right around the corner and will support solving captchas in Firefox & Webkit as well. :-)

i tried but it doesn't work with cloudflare hcaptcha. Nothing happen after await page.solveRecaptchas()
